src/AllowList.php

```diff
@@ -92,6 +96,18 @@ public function getList()
      */
     public function filter(mixed $value): mixed
     {
-        return in_array($value, $this->getList(), $this->getStrict()) ? $value : null;
+        return in_array($value, $this->list, $this->strict) ? $value : null;
+    }
+
+    /**
+     * Will return $value if its present in the allow-list. If $value is rejected then it will return null.
+     *
+     * @template T
+     * @param T $value
+     * @return T|null
+     */
+    public function __invoke(mixed $value): mixed
```

Comment:
It's part of the `FilterInterface` contract that all filters can be invoked along with also calling `filter()` - not sure how bad of a BC break removing it would be.

Reply:
Hm, most probably something which is used in laminas-mvc for "convenience" in laminas-view templates...
Lets keep it as it is just a proxy anyways.
Was just curious.